src/us/kbase/genbank/GbkParser.java

```diff
@@ -172,28 +166,28 @@ public static void main(String[] args) throws Exception {
         final String filename = "Pseudomonas_stutzeri_DSM_10701.gb";
         BufferedReader br = new BufferedReader(new FileReader(new File("test/" + filename)));
         parse(br, new GbkParsingParams(false), filename, new GbkCallback() {
-            @Override
-            public void setGenomeTrackFile(String contigName, String genomeName, int taxId, String plasmid, String filename) throws Exception {
-                pw.println("setGenome: contigName=" + contigName + ", genomeName=" + genomeName + ", taxId=" + taxId + ", plasmid=" + plasmid);
-            }
+		@Override

```

Comment:
Looks like a bug in the way emacs handles Java indentation.  Will patch & re-commit.